apps/style/applab/style.scss

```diff
@@ -8,10 +8,8 @@
 
 $root: '/blockly/media/applab/'; //TODO: Parameterize for asset pipeline
 
-// ** START OF RULES THAT ARE MODIFIED AT RUNTIME IN adjustAppSizeStyles() **
-
 div#visualization {
-  height: 400px;
+  height: 562.5px;
```

Comment:
yes, scss allows doing that math in place, e.g.`scale(0.875 * 1.25)`, or presumably a variable like `$scale_factor` could be multiplied as well.